beacon-chain/blockchain/service.go

```diff
@@ -226,6 +196,20 @@ func (c *ChainService) updateHead(slot uint64) {
 	c.candidateCrystallizedState = nilCrystallizedState
 }
 
+func (c *ChainService) doesPoWBlockExist(block *types.Block) bool {
+	if c.web3Service == nil {
+		return true
```

Comment:
this should be `false`

Reply:
This defaults to true so that nodes that aren't connected to a PoW chain can still validate blocks. I'll leave some comments to clarify.

Reply:
if nodes are not connected to a POW chain, they should not be able to validate a block , no ? is there any situation where a unconnected node should be still able to validate blocks ?

Reply:
Sounds good, in that case you could just add a small note there so that we remember to remove it later

Reply:
I just got rid of the check. For testing purposes, `geth --dev` provides a simple setup.